plugins/calibre.koplugin/search.lua

```diff
@@ -180,7 +180,7 @@ local CalibreSearch = InputContainer:new{
 
 function CalibreSearch:ShowSearch()
     self.search_dialog = InputDialog:new{
-        title = _("Search books"),
+        title = _("Enter string to search for"),
```

Comment:
I prefer to keep programlish like `string` to the backend.

Reply:
Since you like consistency ;-)
```suggestion
        title = _("Enter a word or phrase to look up"),
```

Reply:
Seems 'look up' is not much applicable for calibre.
Maybe, return metadata?
`Enter text to search in metadata`
`Enter words to search in metadata`
Or get rid of "Enter...", just
`Calibre metadata search`

Reply:
> Calibre metadata search

Sounds good to me, but pinging @NiLuJe since I don't use calibre.

Reply:
Yep, sounds good!

Reply:
(for easy commit)
```suggestion
        title = _("Calibre metadata search"),
```